app/main.py

```diff
@@ -2,5 +2,18 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def inner(*args, **kwargs) -> None:
```

Comment:
The `inner` function should return the same type as the `func` it wraps. Currently, it is defined to return `None`, which is incorrect. You should update the return type hint to match the return type of `func`, which is `Callable`.